addons/message_whats_new_v2.11/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "api_version": "0.1",
+  "id": "message_whats_new_v2.11",
+  "name": "What's new message",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.11.0",
+    "max_client_version": "2.11.9"
+  },  
```

Comment:
Is this okay? making `max_client_version` 2.12.0 doesn't make much sense since there will be a new message for 2.12. No need for 2 what's new messages right?

Reply:
Right. This makes sense although I wonder if it should `2.11.1` in case there is dot release and we need to show something else. @bakulf what do you think?

Reply:
The addons could provide a "version" condition instead, that would probably be better for this type of use case. I created a ticket: https://mozilla-hub.atlassian.net/browse/VPN-3179

Reply:
Nahhh, my first comment is wrong ignore me. It should be right though, that's a bug! :bug: